lektor/admin/static/js/views/EditPage.jsx

```diff
@@ -225,6 +248,8 @@ class EditPage extends RecordEditComponent {
         <div className='actions'>
           <button type='submit' className='btn btn-primary'
             onClick={this.saveChanges.bind(this)}>{i18n.trans('SAVE_CHANGES')}</button>
+          <button type='submit' className='btn btn-primary'
```

Comment:
Should both buttons be marked as primary?

Reply:
Is there a different CSS selection that should be used here? My logic here was "they both save - which is the primary function - but one refreshes the page, one takes you to preview"; so I felt that leaving the styling the same would make that evident. If not, I can change it - no worries!

Reply:
I think you can use bootstraps `btn-secondary` for one of them. I think it's best to have the least surprising (or most common) action be marked `btn-primary`. That way `btn-primary` means: *"This is likely what you want"*.

The real question which is most sensible to promote as default. I'm leaning slightly towards saving and not loading the preview as the primary choice.

Reply:
I have made the requested changes here and made it a secondary button.